identity-apps-core/apps/authentication-portal/src/main/webapp/backup-code.jsp

```diff
@@ -245,7 +245,7 @@
                                 <div class="field text-left">
                                     <% String multiOptionURI = request.getParameter("multiOptionURI");
                                         if (multiOptionURI != null &&
-                                            AuthenticationEndpointUtil.isValidURL(multiOptionURI)) {
+                                            AuthenticationEndpointUtil.isValidMultiOptionURI(multiOptionURI)) {
```

Comment:
Is this a new function? Please check if you need to bump the `framework` version inside `identity-apps`.